Modules/Devices/src/Devices.Application/Clients/Queries/ListClients/Handler.cs

```diff
@@ -14,12 +14,15 @@ public Handler(IOAuthClientsRepository oAuthClientsRepository, IMapper mapper)
         _oAuthClientsRepository = oAuthClientsRepository;
         _mapper = mapper;
     }
-    public async Task<IEnumerable<ClientDTO>> Handle(ListClientsQuery request, CancellationToken cancellationToken)
+    public async Task<ListClientsResponse> Handle(ListClientsQuery request, CancellationToken cancellationToken)
     {
         var client = await _oAuthClientsRepository.FindAll(cancellationToken);
```

Comment:
I'm feeling stupid for telling you this, but since you make the same mistake over and over again, I feel like I have to... 

So if a variable contains a list/an array or something else that contains multiple objects, the name of the variable should be **plural**.

```suggestion
        var clients = await _oAuthClientsRepository.FindAll(cancellationToken);
```

(I know that it wasn't part of this change, but it's still your work)